kubert/src/runtime.rs

```diff
@@ -97,6 +136,46 @@ struct LogSettings {
     format: LogFormat,
 }
 
+#[cfg(feature = "prometheus-client")]
+impl ResourceWatchMetrics {
+    fn new(registry: &mut Registry) -> Self {
+        let watch_applies = Family::default();
+        registry.register(
+            "resource_watch_applies",
+            "Count of apply events for a resource watch",
+            watch_applies.clone(),
+        );
+
+        let watch_restarts = Family::default();
+        registry.register(
+            "resource_watch_restarts",
+            "Count of restart events for a resource watch",
+            watch_restarts.clone(),
+        );
+
+        let watch_deletes = Family::default();
+        registry.register(
+            "resource_watch_deletes",
+            "Count of delete events for a resource watch",
+            watch_deletes.clone(),
+        );
+
+        let watch_errors = Family::default();
+        registry.register(
+            "resource_watch_errors",
+            "Count of errors for a resource watch",
+            watch_errors.clone(),
+        );
```

Comment:
```suggestion
        registry.register(
            "applies",
            "Count of apply events for a resource watch",
            watch_applies.clone(),
        );

        let watch_restarts = Family::default();
        registry.register(
            "restarts",
            "Count of restart events for a resource watch",
            watch_restarts.clone(),
        );

        let watch_deletes = Family::default();
        registry.register(
            "deletes",
            "Count of delete events for a resource watch",
            watch_deletes.clone(),
        );

        let watch_errors = Family::default();
        registry.register(
            "errors",
            "Count of errors for a resource watch",
            watch_errors.clone(),
        );
```

Rather than encode the same prefix, I prefer extracting this out into the provided registry so that the local code doesn't have to know about it.